src/build/env.py

```diff
@@ -6,11 +6,16 @@
 import sysconfig
 import tempfile
 import types
-from typing import Dict, Iterable, List, Optional, Sequence, Type
+from typing import Dict, Iterable, List, Optional, Sequence, Type, cast
```

Comment:
I strongly prefer having `typing.cast` instead of `cast`. I generally only use partial imports for `typing` types.

Reply:
Considering all other elements from typing are directly imported to me feels strange to have cast be imported separately. Strongly...